autoload/neomake/makers/ft/haskell.vim

```diff
@@ -1,5 +1,6 @@
 
 function! neomake#makers#ft#haskell#MakerAvailable(command)
+    let s:stack_is_executable = 0
```

Comment:
But I think doing so for just `s:makers`, and then using `unlet! s:makers` to distinguish "not initialized" from "empty list, because nothing was found" should be enough.

The `executable` call is rather cheap, and you'll use `s:makers` then anyway.  `stack exec` should be avoided.